test/api/detection_test.cpp

```diff
@@ -386,9 +386,10 @@ TEST(junction_detection, analyze_sa_tag)
     cmd_arguments args{std::filesystem::path{},
                        std::filesystem::path{},
                        std::filesystem::path{},
-                       "GENOTYPE",
+                       "MYSAMPLE",
                        std::filesystem::path{},
                        std::filesystem::path{},
+                       1,
                        std::vector<detection_methods>{cigar_string, split_read, read_pairs, read_depth},
                        simple_clustering,
                        sVirl_refinement_method,
```

Comment:
I'm just now seeing this and it has nothing to do with this PR, but many of these `cmd_arguments` in the tests (detection_test and input_file_test) are missing some parameters at the end. Even though these apparently fall back to the default values we should probably set them explicitly.

Reply:
Very good comment! I have completely adjusted the args.